threads-api/src/threads-api.ts

```diff
@@ -1006,7 +1147,7 @@ export class ThreadsAPI {
     if (this.verbose) {
       console.debug('[FOLLOW]', res.data);
     }
-    return res.data;
+    return res.data.status === 'ok';
```

Comment:
Reverting those to return `res.data` — the `friendship` response can really come in handy :)